cdap-api/src/main/java/co/cask/cdap/api/data/DatasetContext.java

```diff
@@ -45,6 +45,21 @@
   /**
    * Get an instance of the specified Dataset.
    *
+   * @param namespace The namespace of the Dataset
+   * @param name The name of the Dataset
+   * @param <T> The type of the Dataset
+   * @return An instance of the specified Dataset, never null.
+   * @throws DatasetInstantiationException If the Dataset cannot be instantiated: its class
+   *         cannot be loaded; the default constructor throws an exception; or the Dataset
+   *         cannot be opened (for example, one of the underlying tables in the DataFabric
+   *         cannot be accessed).
+   */
+  <T extends Dataset> T getDataset(String namespace, String name)
+    throws DatasetInstantiationException;
```

Comment:
Changed. Looks like all the functions in this Interface use two-short-line style. Changed all of them.